test/functionalTest/harnessFunctions.sh

```diff
@@ -153,7 +198,22 @@ function dbDrop()
   fi
 }
 
+# ------------------------------------------------------------------------------
+#
+# dbDropTenant - 
+#
+function dbDropTenant()
+{
+  db=$1
```

Comment:
Here we have a problem also, the second param is not taken into consideration.

    if [ $# == 2 ]
    then 
      db=${db}$2
    fi

Something like that would fix it.
And ... this can be done in the 'old' dbDrop, right?
We don't need a new function for an optional second variable, I would say